config/initializers/bullet_train.rb

```diff
@@ -1,6 +1,8 @@
 BulletTrain.configure do |config|
-  # Uncomment this line if you want to bypass strong passwords.
-  # config.strong_passwords = false
+  # Ensures that only strong passwords are used when registering a user.
```

Comment:
```suggestion
  # Use strong passwords when registering a user in non-development environment.
```

Trying to make this wording a little clearer.